scripts/image

```diff
@@ -399,6 +399,8 @@ if [ "$1" = "release" -o "$1" = "mkimage" -o "$1" = "amlpkg" -o "$1" = "noobs" ]
     echo "Creating Amlogic ZIP auto-install package"
     pushd sign > /dev/null
     echo --update_package=/sdcard/$IMAGE_NAME-update.zip > factory_update_param.aml
+    echo --wipe_data >> factory_update_param.aml
+    echo --wipe_cache >> factory_update_param.aml
```

Comment:
Weren't these removed in #575 ? Also after these added we don't need cacb06ec6d4260e4e27ce99a88f0e90d9596d0cc for WeTek Core.